group_vars/all.yml

```diff
@@ -1,12 +1,15 @@
 ---
 
-stockpile_user: user
-local_remote_user: user
-host_remote_user: user
+stockpile_user: root
+local_remote_user: root
+host_remote_user: root
 stockpile_output_path: /tmp/stockpile.json
 container_cli: docker
-# This directory is used to store json
+# The temporary directory is used to store json
 # files generated by backpack containers if applicable
 backpack_output_dir: /tmp/container
 # The user to connecect to kubernetes hosts with
-container_remote_user: user
+container_remote_user: root
+
+# The namespace for the backpack pod to run in
+operator_namespace: "{{ operator_namespace | default('backpack') }}"
```

Comment:
This is specific for kubernetes group right, so shouldn't it be in the file/directory of kubernetes ? Not a hard ask, but it can be confusing given that k8s_pods and so on other roles exist which are run on the instance group. We can also do this in a subsequent pr but just pointing it out here.

Reply:
valid. Updated w/ group_vars/kubernetes.yml and broke the kubernetes specific vars out to there.